api-schema/basic.js test fails due to missing schema file

**Describe the bug**

`npm test` fails in "test/integration/api-schema/basic.js" because this recent apm-server.git change (https://github.com/elastic/apm-server/commit/bcf6752bb8174472acc2190deb475971553826e4#diff-2a4f2dae08baf4f3b2e1fc5153552185R70) added a reference to "rum_experience.json" that isn't being downloaded by "download-json-schemas.sh".

**To Reproduce**

Steps to reproduce the behavior: 

1. Run `npm test`.

```
$ npm test
...
running (cwd: ./): node --unhandled-rejections=strict test/integration/api-schema/basic.js
downloading schemas from GitHub to /tmp/.schemacache...
processing /tmp/.schemacache/metadata.json...
processing /tmp/.schemacache/transactions/transaction.json...
processing /tmp/.schemacache/spans/span.json...
processing /tmp/.schemacache/errors/error.json...
/home/trentm/elastic/apm-agent-nodejs/test/integration/api-schema/basic.js:20
  if (err) throw err
           ^

ResolverError: Error opening file "/tmp/.schemacache/rum_experience.json"
ENOENT: no such file or directory, open '/tmp/.schemacache/rum_experience.json'
    at ReadFileContext.callback (/home/trentm/elastic/apm-agent-nodejs/node_modules/@apidevtools/json-schema-ref-parser/lib/resolvers/file.js:52:20)
    at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:264:13) {
  code: 'ERESOLVER',
  source: '/tmp/.schemacache/rum_experience.json',
  path: null,
  toJSON: [Function: toJSON],
  ioErrorCode: 'ENOENT',
  [Symbol(nodejs.util.inspect.custom)]: [Function: inspect]
}
/home/trentm/elastic/apm-agent-nodejs/test/test.js:129
    if (err) throw err
             ^

Error: non-zero error code
    at ChildProcess.<anonymous> (/home/trentm/elastic/apm-agent-nodejs/test/test.js:37:19)
    at ChildProcess.emit (events.js:314:20)
    at maybeClose (internal/child_process.js:1051:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5) {
  code: 'ENONZERO',
  exitCode: 1
}
```

**Expected behavior**

The test case should pass -- easily fixed by updating "download-json-schemas.sh" to download that missing schema file.


**Environment (please complete the following information)**

- OS: Linux
- Node.js version: v14.8.0
- APM Server version: N/A
- Agent version: latest master (commit 6e487e962d5b488aa6a448a9e64bec9d25fc0915)

**How are you starting the agent? (please tick one of the boxes)**

N/A

--- a/test/integration/api-schema/download-json-schemas.sh
+++ b/test/integration/api-schema/download-json-schemas.sh
@@ -35,6 +35,7 @@ FILES=( \
   "metadata.json" \
   "process.json" \
   "request.json" \
+  "rum_experience.json" \
   "service.json" \
   "span_subtype.json" \
   "span_type.json" \

